lua/autorun/server/sv_cfc_contact_forms.lua

```diff
@@ -3,21 +3,21 @@ util.AddNetworkString( 'CFC_SubmitFeedbackForm' )
 util.AddNetworkString( 'CFC_SubmitBugReport' )
 util.AddNetworkString( 'CFC_SubmitPlayerReport' )
 
-local FORM_PROCESSOR_URL = file.Read( "cfc/contact/url.txt", "DATA" )
-FORM_PROCESSOR_URL = string.Replace(FORM_PROCESSOR_URL, "\r", "")
-FORM_PROCESSOR_URL = string.Replace(FORM_PROCESSOR_URL, "\n", "")
+local FORM_PROCESSOR_URL = file.Read( 'cfc/contact/url.txt', 'DATA' )
+FORM_PROCESSOR_URL = string.Replace( FORM_PROCESSOR_URL, '\r', '' )
+FORM_PROCESSOR_URL = string.Replace( FORM_PROCESSOR_URL, '\n', '' )
```

Comment:
Why are we preferring single quotations?

Reply:
The only lua guide about quotes is "use one or the other, but not both". in this file we used singles more than we used doubles so it was easier to change the doubles to singles